tests/default.nf.test.snap

```diff
@@ -83,8 +83,7 @@
             ],
             [
                 "genome_transcriptome.fasta:md5,d41d8cd98f00b204e9800998ecf8427e",
-                "genome_transcriptome.gtf:md5,d41d8cd98f00b204e9800998ecf8427e",
-                "multiqc_plots:md5,d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
Why is multiqc plots md5 removed ?

Reply:
So this is a stub test, and it was actually a file before but now the module has been fixed upstream, and it's a proper folder, and so no longer getting captured by nf-test, hence the removal